docs/slc_revocation_spec.md

```diff
@@ -0,0 +1,59 @@
+# Short-Lived Credential (SLC) Revocation Specification
```

Comment:
There are quite a few docs here. How about creating a directory for MSIv2 and adding a short index.md to organize everything?